File: Day_1/CPP/Que4.cpp
#include <iostream>
using namespace std;
int main()
{
    int num;
    cout << "Enter number: ";
    cin >> num;
    bool isPrime = true;
    for (int i = 2; i < num / 2; i++)
    {
        if (num % i == 0)
        {
            isPrime = false;
            break;
        }
    }
    if (isPrime)
    {
        cout << "Prime";
    }
    else
    {
        cout << "Not prime";
    }

    return 0;
}

File: Day_1/Java/Que4.java
public class Que4 {
    public static void main(String[] args) {
        int num = 20;
        boolean isPrime = true;
        for (int i = 2; i < num; i++) {
            if (num % i == 0) {
                isPrime = false;
                break;
            }
        }
        if (isPrime) {
            System.out.println("prime");
        } else {
            System.out.println(" not prime");
        }

    }

}
